### DISCUSSION TITLE
Standup meetings

### DISCUSSION THREAD

**Discussion by chrisgnicholas** [Chris-Nicholas-collaborators/EEP153_team_Malthus#4] [Show and tell]
1/23/2024

Chris - github repo established
Bolor - have datahub, github accounts; need guidance on integration
Sam -  up and running on VSCode
Angela - have datahub, github accounts; need guidance on integration
Tia - have datahub, github accounts; need guidance on integration
Steven - up and running on VS Code and datahub

**Comment by chrisgnicholas** [comment]
1/26/2024

- Chris: sketched out basics of population() function; plan to check in my branch this weekend; no blockers

- Bolor: scoping out statistics; needs help github <-> datahub integration

- Sam: environment set up; created graphing notebook of Namibia and South Africa; no blockers

- Angela: environment set up; looked at different indicators; needs help github <-> datahub integration

- Tia: environment set up; needs some help with github<->datahub integration.  Looked at African development indicator(source 11); made graphs of South Africa vs Namibia; noticed a spike in aid in 2004; gave some thought to per and post independence

- Steven: not in attendance

- added new team member at the end; need environment setup

objectives this lab:
- get people up and comfortable with git branches and PRs with datahub.
- continue discussion of relevant stories and indicators

**Comment by chrisgnicholas** [comment]
1/30/2024

Chris - completed population function, dataframe, and raised a PR to Sam

Bolor - read through docs, did research; still blocked on environment setup

Sam - got data pyramids working; working on a PR

Angela - issue with Anaconda; might have to move to Datahub

Kasper - welcome!  won't contribute to Project1; catch up. Has access to github, needs to get setup on datahub

**Reply by t-pappas** [reply]
Tia 1/30
- I looked through the indicators (from wb source 2)
- read thru how to do the unit tests, will probably be more effective to have the code to play around with to figure out how to actually unit test

**Comment by chrisgnicholas** [comment]
(Groundhog day!)

Chris - merged Tia's testing code with mine; merged to main branch

Bolor - just getting started on our presentation

Sam - started a doc for our presentation

Angela - started her branch for experimentation in VSCode, but will join in other effort

Steven - worked on Ag stats, working on animation of population pyramids

Tia - did testing,  merge with Chris

**Comment by chrisgnicholas** [comment]
2/6/2024

Bolor - working on presentation - first draft, going through wbdata documentation

Steven - still experimenting with animated pyramids

Tia - reading through code, other indicators (tourism, mining, aids, etc...)

Sam - put together first draft of presentation

Kasper - catching up

Chris - still fighting GDAL, geopandas installations; doing code reviews
